extensions/2.0/Vendor/EXT_mesh_manifold/README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Status
 
-Draft
+Complete - open source I/O available [here](https://github.com/elalish/manifold/blob/master/bindings/wasm/examples/manifold-gltf.ts).
```

Comment:
Please add a "Reference" section at the end of the document and put the implementation link there.

Reply:
Done.